pkg/scheduler/actions/common/solvers/by_pod_solver.go

```diff
@@ -17,11 +18,7 @@ import (
 	"github.com/NVIDIA/KAI-scheduler/pkg/scheduler/log"
 )
 
-type SolutionValidator func(
-	pendingJob *podgroup_info.PodGroupInfo,
-	victimJobs []*podgroup_info.PodGroupInfo,
-	victimTasks []*pod_info.PodInfo,
-) bool
+type SolutionValidator func(scenario api.ScenarioInfo) bool
```

Comment:
Would it be better for these functions to pass a pointer than by value? All the underlying functions return pointers to the original objects anyways.

Reply:
It's an interface so I think it's by reference implicitly in go